### ISSUE TITLE
Q: For what languages do we want to collect data for?

### ISSUE THREAD

**Issue by omarsar** [dair-ai/research_emotion_analysis#5]
Please include the languages that you think we should collect data for. If you have experience working in a specific language, that will be useful and you can propose collecting emotion-related data in that language.

**Comment by omarsar** [comment]
I have worked with both English and Spanish. I am also looking at my dialect, Creole.

**Comment by fmplaza** [comment]
In my PhD I'm working with both English and Spanish too, but I focus more on Spanish as it is my mother tongue. I have experienced in collecting Twitter messages.

**Comment by maraimm** [comment]
I will contribute for Arabic

Maybe it is good to discuss the data collection in the next meeting. Are we creating new resources or make use of the existing ones?

**Comment by KhalidAlt** [comment]
I would like to contribute in both Arabic and English.

**Comment by omarsar** [comment]
> In my PhD I'm working with both English and Spanish too, but I focus more on Spanish as it is my mother tongue. I have experienced in collecting Twitter messages.

@fmplaza do you know of any large-scale dataset for Spanish? I haven't come across any.

**Comment by omarsar** [comment]
> I will contribute for Arabic
> 
> Maybe it is good to discuss the data collection in the next meeting. Are we creating new resources or make use of the existing ones?

@maraimm we are creating new resources. I will emphasize on the data collection part next meeting. Thanks. Arabic data will be great as well. Have you looked around to see if there any existing datasets for emotion recognition?

Maybe @KhalidAlt feel free to share any information you come across. 

Let's have some updates on this for our next meeting.

**Comment by KwasiArhin** [comment]
I will look up to see if there are any datasets with TWI that i can find.. other I can only participate with English haha

**Comment by franfj** [comment]
I would like to participate in this project. I can work on English and Spanish datasets. Also, I have some experience retrieving data from Twitter (using Python/tweepy mainly), and doing web scraping.

**Comment by fmplaza** [comment]
> > In my PhD I'm working with both English and Spanish too, but I focus more on Spanish as it is my mother tongue. I have experienced in collecting Twitter messages.
> 
> @fmplaza do you know of any large-scale dataset for Spanish? I haven't come across any.

@omarsar I know three different emotion datasets for Spanish labeled at tweet level but they don't include a large data set:

1. [EmoEvent: A Multilingual Emotion Corpus based on different Events.](https://www.aclweb.org/anthology/2020.lrec-1.186.pdf) 
I'm one of the authors of this paper, it has been recently published in the LREC conference. 
The **Spanish version of EmoEvent dataset** contains **8,409 tweets**. Labels: anger, fear, sadness, joy, disgust, surprise, other.

2. [Datasets from SemEval-2018 Task 1: Affect in Tweets](https://www.aclweb.org/anthology/S18-1001.pdf
) **AIT dataset** comprises the datasets used in two subtasks: 

    1. E-c Multi-Label Classification. The dataset contains **7,094 tweets** but it is a Multi-Label Classification Dataset. Labels: anger, anticipation, disgust, fear, joy, love, optimism, pessimism, sadness, surprise, trust, neutral, or no emotion.

    2. EI-oc (emotion intensity ordinal classification) and EI-reg (emotion intensity regression) subtasks.  The dataset contains **7,953 tweets**. Labels: anger, fear, sadness, joy.

**Comment by maraimm** [comment]
For Arabic, there are many efforts and most of them result in small-sized datasets: The following are the datasets I found in the first phase of the search. 

- [AETD dataset - Emotional-Tone](https://github.com/AmrMehasseb/Emotional-Tone) - Type: Tweets - Size: 10065 - Lang: Arabic-Egyptian 
- [SemEval-2018 Task 1: Affect in Tweets (AIT-2018)](https://competitions.codalab.org/competitions/17751)

- [Lama-dataset](https://github.com/UBC-NLP/ara_emotion_naacl2018) - available by contacting the author

- [Emotion in the Headlines](https://ieee-dataport.org/documents/arabic-news-headlines)

**Comment by omarsar** [comment]
@maraimm those are great findings. Do you mind giving us a short overview of your findings in the next meeting? It doesn't have to be a long presentation. We would just like an update.

**Comment by maraimm** [comment]
@omarsar Yeah, sure. I am not sure when is the next meeting, date and time?

**Comment by omarsar** [comment]
@maraimm it's scheduled for next Saturday (25 July 2020 - 15:00 CEST). I will send the zoom link in our Slack group.

**Comment by maraimm** [comment]
Thanks, @omarsar. Unfortunately, I am not sure I will be able to join the call on Saturday. In case, I was not able, shall I prepare something today to share it with the team tomorrow? (Summary for example) 

Will the session be recorded?

**Comment by omarsar** [comment]
@maraimm the summary would be excellent. If it's a recording even better then I can share it with the group when we meet again. All sessions are being recorded.

**Comment by maraimm** [comment]
Hi @omarsar,

I emailed you a short recording.

Thanks

**Comment by omarsar** [comment]
@maraimm Thank you for the video recording. I have added it to our meeting notes.

**Comment by cahya-wirawan** [comment]
Hi,
Sorry for coming late. I found a paper from late 2018 about emotion classification on indonesian twitter: https://www.researchgate.net/publication/330674171_Emotion_Classification_on_Indonesian_Twitter_Dataset
They collected and annotated 7500 tweets with 5 emotions: love, joy, anger, sadness, and fear.

**Comment by rfazeli** [comment]
I can collect data for Persian
